.nycrc

```diff
@@ -1,13 +1,3 @@
 {
-  "reporter": [
-    "lcov",
-    "text"
-  ],
-  "include": [
-    "src/**/*.ts"
-  ],
-  "extension": [
-    ".ts"
-  ],
-  "all": true
+  "extends": "./node_modules/ethereumjs-config/typescript/nyc.json"
```

Comment:
Can we extend from something like `@ethereumjs/config` (or even just `ethereumjs-config/typescript/nyc.json`)?

Reply:
@krzkaczor Would the typescript 3.2 `tsconfig.json inheritance via Node.js packages` work in this case? (sorry I come after the battle). It looks after a `tsconfig.json` file inside the package.
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-2.html

Reply:
Yes, it should work but last time i worked on it it wasn't released yet :D But anyway we are gonna migrate to monorepo for config. And this will solve this problem